### ISSUE TITLE
Migrate support to mysql

### ISSUE THREAD

**Issue by jneto81** [fragmenta/fragmenta-cms#3]
the sql script don't work with mysql

**Comment by kennygrant** [comment]
Yes, both the migrate and backup/restore commands at present call psql, rather than being database agnostic. I need to consider which user to call commands with as well, as for initial bootstrap we can't rely on the config db user as there is none.
